config/trackers.go

```diff
@@ -13,3 +13,8 @@ var Trackers = []string{
 	"udp://tracker.internetwarriors.net:1337/announce",
 	"http://mgtracker.org:6969/announce",
 	"http://tracker.baka-sub.cf/announce"}
+
+// NeededTrackers : Array indexes of Trackers for needed tracker in a torrent file
```

Comment:
No that's okay. Good to ask :). Can you reply to me for the tracker regex? I don't know if I should check for "&" or not. If I have to, we need to convert "&" characters then because how do magnet will work?